formatTest/unit_tests/expected_output/if.re

```diff
@@ -84,13 +70,7 @@ if printIfFirstArgGreater {
   thisDoesnt even have2 be simple
 };
 
-if (
-  if x {
-    true
-  } else {
-    false
-  }
-) {
+if (if x {true} else {false}) {
```

Comment:
I'm pretty neutral on this, besides that an if in an if test is a bit confusing.